EIPS/eip-3.md

```diff
@@ -42,6 +42,8 @@ The opcode `CALLDEPTH` should return the remaining call stack depth. A value of
 
 The actual call stack depth, as well as the call stack depth limit, are present in the EVM during execution, but just not available within the EVM. The implementation should be fairly simple and would provide a cheap and way to protect against call stack attacks.
 
-# Implementation
+# Abandoned
+
+On Dec 8, 2019, [Brent Allsop](mailto:brent.allsop@canonizer.com) reached out to [Martin Holst Swende](mailto:martin@swende.se) asking about thoughts for the future of this EIP.
+He replied with: "I'd rather just withdraw that one, I think. There's hardly any justification for it any longer."
```

Comment:
```suggestion
# Implementation

Not implemented.
```

Reply:
I am not sure if the new status should be 'Abandoned' because 'Abandoned' EIP, to my understanding is the one that is dropped by the author for some reason and may still hold value. This EIP can be moved ahead in the EIP process to attain 'Accept' or 'Final' status at some point. 

By the response mentioned above, IMO this EIP may be changed to 'Rejected' or 'Superseded' (if that is the case) because 'Withdrawn' is not a status defined in EIP1 unless people would like to propose it.

Reply:
"Rejected" is something ACD decides on. "Superseded" is defined for EIPs which were previously marked "Final". I am not saying these are good or bad, just citing EIP-1.